File: cs/calculator.cs
using System;

namespace Calculator
{
    class Calculation
    {
        static int Main()
        {
            Console.WriteLine("This is an addition calculator.");
            while (true)
            {
                Console.WriteLine("Input your first number, press enter, then input your second number, press enter.");
                checked
                {
                    try
                    {
                        Console.WriteLine($"Your result is: {Convert.ToDecimal(Console.ReadLine()) + Convert.ToDecimal(Console.ReadLine())}");
                        break;
                    }
                    catch (System.FormatException)
                    {
                        Console.WriteLine("Sorry but your input wasn't in the correct format.");
                    }
                }
            }
            return 0;
        }
    }
}

File: python/calculator.py
print("This is an addition calculator")
while True:
    try:
        print("Input your first number, press enter, then input your second number, press enter.")
        conditionOne = float(input())
        conditionTwo = float(input())
        result = conditionOne + conditionTwo
        print("Your result is: {}".format(result))
        break
    except ValueError:
        print("Sorry but your input wasn't in the correct format.")